pages/index.js

```diff
@@ -13,7 +25,43 @@ export default function Home() {
 
       <main>
         <Headline>Tutorial Maker</Headline>
+        <HomeButtons>
+          <Button
+            isPrimary={!session}
+            onClick={() => {
+              if (session) {
+                signOut({
+                  callbackUrl: "/",
+                });
+              } else {
+                signIn();
+              }
+            }}
+          >
+            {session ? (
+              <TbLogout style={iconStyle} />
+            ) : (
+              <TbLogin style={iconStyle} />
+            )}
+            {session ? "Sign out" : "Sign In"}
+          </Button>
```

Comment:
here you can do the same trick I mentioned above.

Reply:
I did. :)